README.md

```diff
@@ -90,15 +90,13 @@ to build.
 1. Install the prerequisites: `npm install`.
 1. Configure the app by copying `config.sample.json` to `config.json` and
    modifying it (see below for details).
-1. `npm run dist` to build a tarball to deploy. Untaring this file will give
+1. Run `npm run dist` to build a tarball to deploy. Untaring this file will give
    a version-specific directory containing all the files that need to go on your
-   web server.
-
-Note that `npm run dist` is not supported on Windows, so Windows users can run `npm
-run build`, which will build all the necessary files into the `webapp`
-directory. The version of Riot will not appear in Settings without
-using the dist script. You can then mount the `webapp` directory on your
-webserver to actually serve up the app, which is entirely static content.
+   web server. (Note that `npm run dist` is not supported on Windows, so Windows
```

Comment:
I really don't think moving this is necessary

Reply:
it makes it easier to read for non-windows users, cause they don't have to read the whole text in brackets

Reply:
I'm pretty certain that as soon as someone reads the word "Windows", they'll stop reading if they don't care about Windows.